deepmd/utils/tabulate.py

```diff
@@ -175,7 +175,24 @@ def build(
         """
         # tabulate range [lower, upper] with stride0 'stride0'
         lower, upper = self._get_env_mat_range(min_nbor_dist)
-        if isinstance(self.descrpt, deepmd.descriptor.DescrptSeA):
+        if isinstance(self.descrpt, deepmd.descriptor.DescrptSeAtten):
```

Comment:
I would also suggest supporting the situation for se_a + type_embedding. They can share the same OP.

Reply:
I closed this conversation as I think it can be supported in a separate PR. It is not resolved.